ocs_ci/ocs/ui/acm_ui.py

```diff
@@ -189,6 +190,55 @@ def navigate_data_services(self):
             )
             raise NoSuchElementException
 
+    def navigate_disaster_recovery(self, timeout=200):
```

Comment:
This function is not needed, you can use existing function `navigate_data_services`